ecosystem/resources/members/dynacir.toml

```
@@ -0,0 +1,8 @@
+name = "dynacir"
+url = "https://github.com/derek-wang-ibm/dynacir"
+description = "Dynacir is a transpilation plugin for Qiskit that optimizes dynamic quantum circuits."
+licence = "Apache License 2.0"
+contact_info = "_No response_"
+alternatives = "_No response_"
+affiliations = "IBM Quantum"
+labels = [ "Optimization", "Prototype", "Transpiler plugin",]
```

```suggestion
labels = [ "Optimization", "Prototype",]
group = "transpiler_plugin"
```

The issue template is a bit outdated